src/main/java/dev/snowdrop/release/App.java

```diff
@@ -270,7 +271,9 @@ public void setupRelease(
 
         GitConfig cpaasConfigGitConfig = cpaasCfgService.buildGitConfig(release, gluser, gltoken, Optional.of(CPaaSConfigUpdateService.CPAAS_REPO_NAME));
         git.initRepository(cpaasConfigGitConfig);
-        cpaasCfgService.newRelease(cpaasConfigGitConfig, release, false);
+        if (!release.isTestMode()) {
+            cpaasCfgService.newRelease(cpaasConfigGitConfig, release, false);
+        }
```

Comment:
We can check the test mode in the `newRelease` method, right?

Reply:
Since the `release` is already a parameter, which includes a flat indicating it's a test release, I've moved that if into the `newRelease` method.